Resources/public/templates/fields/edit/checkbox.hbt

```diff
@@ -1,4 +1,4 @@
-<div class="pure-g ez-editfield-row">
+<div class="pure-g ez-editfield-row {{#if isNotTranslatable}}ez-editfield-not-translatable{{/if}}">
```

Comment:
I would add this class on the view container instead. Also, given it represents a state, it should be name `is-not-translatable` (or something like that depending on the name of this state)